server/src/main/java/org/elasticsearch/health/HealthPeriodicLogger.java

```diff
@@ -311,7 +312,7 @@ boolean tryToLogHealth() {
                 RunOnce release = new RunOnce(currentlyRunning::release);
                 try {
                     ActionListener<List<HealthIndicatorResult>> listenerWithRelease = ActionListener.runAfter(resultsListener, release);
-                    this.healthService.getHealth(this.client, null, false, 0, listenerWithRelease);
+                    this.healthService.getHealth(this.client, null, true, 0, listenerWithRelease);
```

Comment:
We should discuss whether we're good with this change in particular

Reply:
Discussed offline. @dakrone to do some analysis to see how computationally expensive this. I'm +1 doing this though if it's at a reasonable cost.
Another idea that had been floated in the past is doing the first API call and then making a second API call with `verbose=true` if there are any red indicators.